src/api/index.tsx

```diff
@@ -185,9 +197,7 @@ export const callApi = async <SuccessResponseType extends {}>({
   } catch (error) {
     log.debug('Error caught in callApi():', error);
 
-    return {
-      error,
-    };
+    return { error };
```

Comment:
No, not enforcing whitespace is a known (and intentional) limitation to prettier. I only fixed it because it bothered me 😎 but I wouldn't have done so if I were not already touching this code.